README.md

```diff
@@ -3,4 +3,4 @@ buttonmen
 
 Buttonmen - an online dice game
 
-Development forum: http://z13.invisionfree.com/ButtonMenOnlineDev/
+Come and play with us at http://www.buttonweavers.com/
```

Comment:
Suggest rewording a little, maybe to "Play Buttonmen at http://www.buttonweavers.com/"

I don't feel strongly; overall this change seems fine.